pipelines/defaults.json

```diff
@@ -53,5 +53,6 @@
             "numMachines"    : ["3"]
         }
     },
-    "defaultsUrl"            : "https://raw.githubusercontent.com/adoptium/ci-jenkins-pipelines/master/pipelines/defaults.json"
+    "defaultsUrl"            : "https://raw.githubusercontent.com/adoptium/ci-jenkins-pipelines/master/pipelines/defaults.json",
+    "signNodeLabel"            : "eclipse-codesign"
```

Comment:
Hmm, I wonder if this should be in Adopt defaults json as opposed to defaults json...

Reply:
Both default JSONs' variables were mapped and an if statement was created to ensure default json was used if adopt default shell script wasn't in play, I guess it would be safer to use adopt default, I'll make the change and commit now. thanks for the correction

Reply:
they both seem to be feeding from the same URL, I think it's safe